Mathlib/Analysis/SpecialFunctions/ContinuousFunctionalCalculus/Rpow/Basic.lean

```diff
@@ -689,6 +689,12 @@ omit [T2Space A] [IsTopologicalRing A] in
 lemma _root_.IsStrictlyPositive.rpow {a : A} {y : ℝ} (ha : IsStrictlyPositive a) :
     IsStrictlyPositive (a ^ y) := by grind
 
+omit [T2Space A] [IsTopologicalRing A] in
+theorem _root_.IsStrictlyPositive.commute_iff {a b : A} (ha : IsStrictlyPositive a)
+    (hb : IsStrictlyPositive b) : Commute a b ↔ IsStrictlyPositive (a * b) := by
+  rw [commute_iff_mul_nonneg ha.nonneg hb.nonneg]
+  exact ⟨fun h => ha.isUnit.mul hb.isUnit |>.isStrictlyPositive h, fun h => h.nonneg⟩
```

Comment:
This feels like it should be possible to move it somewhere else, maybe even out of this file?

Perhaps this is here as a vestige from before `commute_iff_mul_nonneg` moved.